.github/workflows/build_and_deploy_docs.yaml

```diff
@@ -0,0 +1,87 @@
+name: Build and Deploy Documentation on Github Pages
+
+on:
+  # Runs when PR is closed and merged
+  pull_request:
+    types:
+      - closed
+
+# Allows you to run this workflow manually from the Actions tab
+workflow_dispatch:
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+    
+jobs:
+  build-documentation:
+    if_merged:
```

Comment:
this is the name of the job, so either `if_merged` or `build_documentations`. Both wouldn't work